src/Core/benchmarks/App.Metrics.Benchmarks/App.Metrics.Benchmarks.csproj

```diff
@@ -4,6 +4,7 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
+    <LangVersion>7.3</LangVersion>
```

Comment:
The `<LangVersion>` must be specified to something specific - otherwise the `BenchmarkDotNet` compiler barfs when it attempts to auto-generate the benchmark programs.